html5/_main_toc.html.slim

```diff
@@ -10,17 +10,17 @@ nav.sidebar-nav
             <path fill-rule="evenodd" clip-rule="evenodd" d="M2.5275 5.62018C2.4075 5.6622 2.30022 5.70321 2.18648 5.74967C1.28587 6.11129 0.757784 6.52907 0.440004 7.11615C0.00100999 7.92929 0 8.90445 0 9.99456C0.00101011 10.2041 0.233739 10.3313 0.409296 10.2176L2.10931 9.12465C1.84668 8.10202 1.99072 6.93837 2.5275 5.62018Z" fill="#D1D5DB"/>
             <path d="M2.16122 9.85376C1.46162 10.2578 1.0198 11.0194 0.863028 11.7982C0.706461 12.5776 1.00586 13.6433 1.41233 13.8778C1.81799 14.1114 2.89032 13.839 3.48689 13.3132C4.08366 12.7879 4.52225 12.0253 4.52225 11.2162L4.09518 10.9699C4.07821 12.0758 2.70628 13.0237 1.94243 12.9596C1.50485 12.3303 1.6394 10.6681 2.58971 10.1L2.16122 9.85376Z" fill="#D1D5DB"/>
           </svg>
-          a.bd-toc-link.bd-toc-link-active.main-link Getting started
+          a.bd-toc-link.main-link Getting started
         <svg class="chevron" width="11" height="6" viewBox="0 0 11 6" fill="none" xmlns="http://www.w3.org/2000/svg">
           <path fill-rule="evenodd" clip-rule="evenodd" d="M10.5024 5.70711C10.1119 6.09763 9.47871 6.09763 9.08818 5.70711L5.7953 2.41421L2.50241 5.70711C2.11188 6.09763 1.47872 6.09763 1.0882 5.70711C0.697672 5.31658 0.697672 4.68342 1.0882 4.29289L5.08819 0.292893C5.47871 -0.0976311 6.11188 -0.0976311 6.5024 0.292893L10.5024 4.29289C10.8929 4.68342 10.8929 5.31658 10.5024 5.70711Z" fill="#D1D5DB"/>
         </svg>
 
-      ul.nav.navbar-nav.active-list
+      ul.nav.navbar-nav
         li
-          a.nav-link href="https://docs.killbill.io/0.24/what_is_kill_bill.html"
+          a.nav-link href="/latest/what_is_kill_bill.html"
```

Comment:
Could you keep `/0.24` instead of `/latest`? This would break the staging website navigation.

We'll do a global replacement once the project is finished. Thanks!